aptos-move/vm-genesis/src/lib.rs

```diff
@@ -399,7 +399,15 @@ fn initialize(
 }
 
 fn initialize_features(session: &mut SessionExt<impl MoveResolver>) {
-    let features: Vec<u64> = vec![1, 2, 5];
+    let features: Vec<u64> = vec![
+        1, // code dependency check
+        2, // treat friend as private
+        3, // SHA2-512 and RIPEMD-160 natives
```

Comment:
Could you replace the flags with `FeatureFlag` enum defined under `types/src/on_chain_config/aptos_features.rs`?

Reply:
Done.